contentScript.js

```diff
@@ -419,9 +441,27 @@ const createForm = () => {
   timeSlotSelector.appendChild(two)
   timeSlotSelector.appendChild(three)
   timeSlotSelector.appendChild(four)
-  let timeslotlabel = createLabel("timeslotinputlabel", timeslotinputid, "Enter time slot preference: ", textLabelStyles);
+  let timeslotlabel = createLabel("timeslotinputlabel", timeslotinputid, "Select time slot preference: ", textLabelStyles);
   let timeslotwarn = createWarningText("If these are not the cases available there, first slot will be selected automatically.", warnLabelStyles);
 
+  let costpreferenceid = "costpreference";
+  let costPreferenceSelector = createSelectInput(costpreferenceid, "", costPreferenceValue)
+  let free = createSelectOptions('select-free', 'Free', 'Free', costPreferenceValue === 'Free')
+  let paid = createSelectOptions('select-paid', 'Paid', 'Paid', costPreferenceValue === 'Paid')
+  costPreferenceSelector.appendChild(free);
+  costPreferenceSelector.appendChild(paid);
+  let costpreferencelabel = createLabel('costpreferencelabel', costpreferenceid, "Select Cost preference", textLabelStyles);
+
+  let vaccinepreferenceid = "vaccinepreference";
+  let vaccinePreferenceSelector = createSelectInput(vaccinepreferenceid, "", vaccinePreferenceValue)
```

Comment:
This way works too. But I was expecting more of checkbox like thing. 
Have you seen the cowin portal, 
![Screenshot 2021-05-22 at 11 29 39 AM](https://user-images.githubusercontent.com/11269256/119216394-14931400-baf1-11eb-8a43-170cf883b299.png)
Here I can select multiple filters

Reply:
Ahh okay, I was going to create an issue later to make something like this
I will do it in this PR 👍